### PR TITLE
Add initial django project

### PR DESCRIPTION
**Pull request by sliverc**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  db:
+    image: postgres
```
Comment (anehx) [inline docker-compose.yml]: Lock version?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+language: python
+
+python:
+  - "3.6"
+
+services:
+  - postgresql
```
Comment (anehx) [inline .travis.yml]: Lock version to be the same as in `docker-compose.yml`

**Comment by sliverc** [comment]
@anehx Locked and fixed CI

File: docker-compose.yml [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
   db:
-    image: postgres
+    image: postgres:9.3
     ports:
       - '5432:5432'
     environment:
-      - POSTGRES_USER=caluma
-      - POSTGRES_PASSWORD=caluma
+      - POSTGRES_USER={{cookiecutter.project_name}}
+      - POSTGRES_PASSWORD={{cookiecutter.project_name}}
```
Comment (anehx) [inline docker-compose.yml]: That's not correct I think...

Comment (sliverc) [reply docker-compose.yml]: Editor was playing fool with me... :) its fixed now.

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+version: '2'
```
Comment (anehx) [inline docker-compose.yml]: Why not the latest version `3`?

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+"""Setuptools package definition."""
+
+from setuptools import find_packages, setup
+
+setup(
+    name='caluma',
+    version='0.0.0',
+    author='Adfinis SyGroup AG',
+    author_email='https://adfinis-sygroup.ch/',
```
Comment (anehx) [inline setup.py]: Are we sure that "Adfinis SyGroup AG" should be the author? Also, this email is not an email address..

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    description='Caluma Service providing GraphQL API ',
+    url='https://adfinis-sygroup.ch/',
+    packages=find_packages(),
```
Comment (anehx) [inline setup.py]: Maybe add our license model here aswell? https://docs.python.org/2/distutils/setupscript.html#additional-meta-data

File: docker-compose.yml (hunk 4 above)
Comment (sliverc) [reply docker-compose.yml]: Fair point. Updated.

File: setup.py (hunk 5 above)
Comment (sliverc) [reply setup.py]: Removed author and author_emai. Not really relevant right now anyway.

File: setup.py (hunk 6 above)
Comment (sliverc) [reply setup.py]: license added.

**Comment by sliverc** [comment]
ready for review again

**Pull request merged**